src/controllers/VATController.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Body, Controller, Get, Post, Put, Query, Request, Response, Route, Security, Tags,
+} from 'tsoa';
+import express from 'express';
+import { body } from 'express-validator';
+import VATService, {
+  VATListResponse,
+  VATParams,
+  VATSummary,
+} from '../services/VATService';
+import { ListParams } from './ListParams';
+import { WrappedApiError } from '../helpers/error';
+import { ValueAddedTax } from '../entity/ValueAddedTax';
+import { validate } from '../helpers/validation';
+
+@Route('VAT')
+@Tags('Value Added Tax')
+export class VATController extends Controller {
```

Comment:
Even in de readme toevoegen dat je handmatig (in de database) VAT entities moet maken. Er is namelijk geen POST endpoint.